### PR TITLE
feat(client): Change credit tile direction text when RFQ accepted

### PR DESCRIPTION
**Pull request by TobyLi96**

### CODE REVIEW

File: src/client/src/App/Credit/CreditRfqs/CreditRfqCards/CardHeader.tsx [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
     const instrument = useCreditInstrumentById(instrumentId)
+    const terminated = isRfqTerminated(rfqState)
+    const accepted = rfqState === RfqState.Closed
 
     return (
       <DirectionContainer direction={direction} terminated={terminated}>
         {direction === Direction.Buy && (
           <DirectionLabel direction={direction} terminated={terminated}>
-            YOU BUY
+            {accepted ? "     YOU BOUGHT" : "YOU BUY"}
```
Comment (ivanfalanga-adaptive) [inline src/client/src/App/Credit/CreditRfqs/CreditRfqCards/CardHeader.tsx:29]:
```suggestion
            YOU {accepted ? "BOUGHT" : "BUY"}
```

**Review by ivanfalanga-adaptive — APPROVED** [review]

File: src/client/src/App/Credit/CreditRfqs/CreditRfqCards/CardHeader.tsx (hunk 1 above)
Comment (TobyLi96) [reply src/client/src/App/Credit/CreditRfqs/CreditRfqCards/CardHeader.tsx]: Changed to using a div

File: src/client/src/App/Credit/CreditRfqs/CreditRfqCards/CardHeader.tsx [hunk 2]
```diff
@@ -1,29 +1,32 @@
...
     const instrument = useCreditInstrumentById(instrumentId)
+    const terminated = isRfqTerminated(rfqState)
+    const accepted = rfqState === RfqState.Closed
 
     return (
       <DirectionContainer direction={direction} terminated={terminated}>
         {direction === Direction.Buy && (
           <DirectionLabel direction={direction} terminated={terminated}>
-            YOU BUY
+            <div>{accepted ? "YOU BOUGHT" : "YOU BUY"}</div>
```
Comment (ivanfalanga-adaptive) [inline src/client/src/App/Credit/CreditRfqs/CreditRfqCards/CardHeader.tsx:29]:
```suggestion
            <div>YOU {accepted ? "BOUGHT" : "BUY"}</div>
```

File: src/client/src/App/Credit/common/RfqTicketHeader.ts [hunk 3]
```diff
@@ -67,6 +68,12 @@ export const DirectionLabel = styled.div<WithDirection & WithTerminated>`
     -ms-transform: skewX(-24deg);
     transform: skewX(-24deg);
   }
+
+  div {
+    width: ${({ direction }) =>
+      direction === Direction.Buy ? "45px" : "50px"};
```
Comment (ivanfalanga-adaptive) [inline src/client/src/App/Credit/common/RfqTicketHeader.ts:74]:
```suggestion
    width: "50px"
```

Comment (ivanfalanga-adaptive) [reply src/client/src/App/Credit/common/RfqTicketHeader.ts:74]: seems like 50 works for both?

Comment (TobyLi96) [reply src/client/src/App/Credit/common/RfqTicketHeader.ts:74]: To me it's slightly off but I guess nobody would care about such a small difference.

**Pull request merged**